alerts.go

```diff
@@ -16,95 +16,115 @@ type AlertsService interface {
 
 var _ AlertsService = (*Client)(nil)
 
-type Alert struct {
+// AlertInternal is an alert with the time.Duration attributes being the string
+// representation. This is how the alerter service expects them.
+type AlertInternal struct {
```

Comment:
Isn't it possible to update the input of the alerting API, to standardize around ints?

Reply:
Yes we probably could do that. You're right it will probably save us some headaches